front/app/components/ProjectAndFolderCards/ProjectAndFolderCardsInner.tsx

```diff
@@ -53,23 +54,20 @@ const ProjectAndFolderCardsInner = ({
   statusCounts,
   showTitle,
   showSearch,
-  search,
   layout,
   publicationStatusFilter,
   onChangeTopics,
   onChangeAreas,
   onChangeTab,
   onChangeSearch,
+  rootLevelOnly,
 }: Props) => {
-  // if a search string is provided, we want to show all depths of admin publication
-  const rootLevelOnly = search && search.length > 0 ? false : true;
 
   const adminPublications = useAdminPublications({
     pageSize: 6,
     publicationStatusFilter,
     rootLevelOnly,
     removeNotAllowedParents: true,
-    search,
```

Comment:
This isn't passed in at first, but set using the `onChangeSearch` hook that comes from `useAdminPublications`